### PR TITLE
Update footer icons on the NTP

### PR DESCRIPTION
**Pull request by zenparsing**
Resolves https://github.com/brave/brave-browser/issues/39421

Screenshot 2024-06-28 at 10 41 14 AM

## Submitter Checklist:

- [ ] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

**Review by mkarolin — APPROVED** [review]
`strings`++

File: components/brave_new_tab_ui/components/default/page/index.tsx [hunk 1]
```diff
@@ -392,11 +392,7 @@ export const App = styled('div') <AppProps & HasImageProps>`
 export const Link = styled('a') <{}>`
   text-decoration: none;
   transition: color 0.15s ease, filter 0.15s ease;
-  color: rgba(255, 255, 255, 0.8);
-
-  &:hover {
-    color: rgba(255, 255, 255, 1);
-  }
+  color: rgb(255, 255, 255);
```
Comment (fallaciousreasoning) [inline components/brave_new_tab_ui/components/default/page/index.tsx:395]: optional:

```suggestion
  color: white;
```

or 

```suggestion
  color: ${color.white};
```

File: components/brave_new_tab_ui/components/default/page/index.tsx [hunk 2]
```diff
@@ -413,9 +409,8 @@ export const PhotoName = styled('div') <{}>`
   align-self: flex-end;
   -webkit-font-smoothing: antialiased;
   box-sizing: border-box;
-  font-size: 12px;
-  font-family: Poppins, sans-serif;
-  color: rgba(255, 255, 255, 0.6);
+  font: ${font.small.regular};
+  color: rgb(255, 255, 255);
```
Comment (fallaciousreasoning) [inline components/brave_new_tab_ui/components/default/page/index.tsx:413]: ditto

File: components/brave_new_tab_ui/components/default/page/index.tsx [hunk 3]
```diff
@@ -482,9 +462,9 @@ export const IconButtonSideText = styled('label') <IconButtonSideTextProps>`
   display: grid;
   grid-template-columns: auto auto;
   align-items: center;
-  margin-right: ${p => p.textDirection === 'ltr' && '24px'};
-  margin-left: ${p => p.textDirection === 'rtl' && '24px'};
-  color: inherit;
+  padding-right: ${p => p.textDirection === 'ltr' && '4px'};
```
Comment (fallaciousreasoning) [inline components/brave_new_tab_ui/components/default/page/index.tsx:465]: optional:

```suggestion
  padding-right: ${p => p.textDirection === 'ltr' && spacing.s};
```

File: components/brave_new_tab_ui/components/default/page/index.tsx [hunk 4]
```diff
@@ -482,9 +462,9 @@ export const IconButtonSideText = styled('label') <IconButtonSideTextProps>`
...
+  padding-left: ${p => p.textDirection === 'rtl' && '4px'};
```
Comment (fallaciousreasoning) [inline components/brave_new_tab_ui/components/default/page/index.tsx:466]: ditto

File: components/brave_new_tab_ui/components/default/page/index.tsx [hunk 5]
```diff
@@ -482,9 +462,9 @@ export const IconButtonSideText = styled('label') <IconButtonSideTextProps>`
...
+  color: rgb(255, 255, 255);
```
Comment (fallaciousreasoning) [inline components/brave_new_tab_ui/components/default/page/index.tsx:467]:
```suggestion
  color: white;
```

File: components/brave_new_tab_ui/components/default/page/index.tsx [hunk 6]
```diff
@@ -497,10 +477,17 @@ export const IconButtonSideText = styled('label') <IconButtonSideTextProps>`
...
-    margin-left: ${p => p.textDirection === 'ltr' && '0'};
-    margin-right: ${p => p.textDirection === 'rtl' && '0'};
+    margin: 0;
+    height: auto;
+    width: auto;
+    padding: 4px;
+    opacity: 1;
     /* No need to show the outline since the parent is handling it */
     outline: 0;
+    --leo-icon-size: 14px;
```
Comment (fallaciousreasoning) [inline components/brave_new_tab_ui/components/default/page/index.tsx:487]: nit: we normally put variable declarations at the top of a CSS block

**Review by fallaciousreasoning — APPROVED** [review]
LGTM % nits

**Pull request merged**